src/test/java/net/revelc/code/impsort/LineEndingEdgeCasesTest.java

```diff
@@ -40,16 +39,15 @@
   private static Grouper eclipseDefaults =
       new Grouper("java.,javax.,org.,com.", "", false, false, true);
 
-  @Rule
-  public TemporaryFolder folder =
-      new TemporaryFolder(new File(System.getProperty("user.dir"), "target"));
+  @TempDir
+  public File folder;
```

Comment:
I'd strongly prefer our temporary files be located under the Maven build directory, `target/`. This could be done with `TemporaryFolder`, but looks like `TempDir` is limited. See junit-team/junit5#2088

This is probably an acceptable change for this project, but if there's something that can be done to keep it in `target/` instead of `/tmp`, I'd be relieved.

Reply:
I couldn't find a way to do it, so I thought I'd post a PR without it and see what the maintainers think, and possibly start a discussion. But I see you merged it already :-)

One thing that might be possible is to override the `java.io.tmpdir` for this test and force `@TempDir` to create it under `target/`. Let me play around with it a bit and see if this is a viable option.

Reply:
@ctubbsii Following up - The JVM reads `java.io.tmpdir` when it initializes the `jdk.internal.util.StaticProperty` class, and `java.nio.file.TempFileHelper` then uses that. 
TL;DR - while it's not hard to overwrite `java.io.tmpdir` during runtime, that won't affect where JUnit's `@TempDir` creates temporary files, so that idea is a bust.

Another direction is to abandon that annotation and have explicit code in a `@BeforeEach` method to create a temp dir under `target` and then tear it down in an `@AfterEach` method. I'm not sure it's worth the need to maintain this code later on, but if you think it is, I'll gladly send a followup patch. Let me know what you think.

Reply:
I think it's possible to set `java.io.tmpdir` in the `maven-surefire-plugin` so it will affect the test that way, but I have not yet verified that.

Reply:
Good point! I didn't think of doing that.
Note that it won't help for the usecase you manually run the test from your IDE, but I guess that's less of a concern.

Here's my attempt at it, let me know if that's what you meant: #52